api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -665,7 +665,7 @@ public void saveGlobalProperty_shouldNotAllowDifferentPropertiesToHaveTheSameStr
 	}
 	
 	/**
-	 * @see {@link AdministrationService#saveGlobalProperties(List<QGlobalProperty;>)}
+	 * @see AdministrationService#saveGlobalProperties(List<QGlobalProperty;>)
```

Comment:
I see a couple of these 'Q's after the opening angel brackets, can you please remove them?